mutiny-core/src/nostr/mod.rs

```diff
@@ -568,12 +608,15 @@ impl<S: MutinyStorage> NostrManager<S> {
             .map(|profile| NostrWalletConnect::new(&context, xprivkey, profile).unwrap())
             .collect();
 
+        let nwc_profile_version = NwcProfileVersion::new();
```

Comment:
this needs to be read from storage, otherwise you'll always be starting with version 0